packages/utils/telemetry-utils/package.json

```diff
@@ -44,7 +56,7 @@
 		"test:coverage": "c8 npm test",
 		"test:mocha": "mocha --ignore \"dist/test/types/*\" --recursive dist/test -r node_modules/@fluidframework/mocha-test-setup",
 		"test:mocha:verbose": "cross-env FLUID_TEST_VERBOSE=1 npm run test:mocha",
```

Comment:
tests were changed to build ESM so this needs updated to `lib/test` instead of `dist/test` x2